ymdk/bface - RGB config and minila layout (#9127)
qmk/qmk_firmware: keyboards/ymdk/bface/keymaps/minila/readme.md

> Basically layout like on Filco Minila board, found here: https://geekboards.ru/product/filco-minila
> There is one single function layer that can be accessed by pressing both FN keys and also by dolding down CAPS LOCK. 
> Enter is aslo a Right contol when held down. Shifts are also '(' and ')' respectevly when tapped.

Reviewer: ```suggestion
Basically layout like on Filco Minila board, found here: https://geekboards.ru/product/filco-minila

There is one single function layer that can be accessed by holding either FN key or Caps Lock. Enter is also a Right Control when held down. Shifts are also '(' and ')' respectively when tapped.

```

You can control underglow RGBs:

> FN+Z - Toggle
> FN+X - Effect
> FN+C - Color
> FN+V - Brightness lower
> FN+B - Brightness higher

Reviewer: ```suggestion

* FN+Z - Toggle
* FN+X - Effect
* FN+C - Color
* FN+V - Brightness lower
* FN+B - Brightness higher
```
